### PR TITLE
added Troubleshooting.md and some extra documentation

### PR DESCRIPTION
**Pull request by Hammarn**
Maybe a bit wordy in some parts. Might also need a spell check; 
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # NGI-RNAseq
+## 1.4dev
```
Comment (ewels) [inline CHANGELOG.md]: Also here: https://github.com/SciLifeLab/NGI-RNAseq/blob/master/main.nf#L75

File: docs/usage.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 # NGI-RNAseq Usage
 
+## General Nextflow info
+Nextflow handles job submissions on SLURM or other enviroments, and supervises the jobs. Thus the Nextflow proccess must run until the pipeline is finished. We recomend that you put the process runnig in the background through `screen`/`tmux` or similar tool. Alternatively submitted as it's own job through your job scheduler.
```
Comment (ewels) [inline docs/usage.md]: "supervises **running** jobs" maybe?

Comment (ewels) [reply docs/usage.md]: Recomend -> Recommend

Comment (ewels) [reply docs/usage.md]: Runnig -> Running

Comment (ewels) [reply docs/usage.md]: > Alternatively submitted as it's own job through your job scheduler.

I think only if running in `local` mode? Probably needs checking...?

**Comment by Hammarn** [comment]
Running a test to make sure that submitting through a sbatch job actually works. Other than that I think I'm done with the updates to this PR.

**Comment by Hammarn** [comment]
Test complete - seems that submitting Nextflow in a sbatch job works just fine.

File: docs/Troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+Note that if your sample name is "messy" then you have to be very perticular with your glob specification. A file name like `L1-1-D-2h_S1_L002_R1_001.fastq.gz` can be difficult enough for a human to read. Specifying `*{1,2}*.gz` wont work give you what you want Whilst `*{R1,R2}*.gz` will. 
+
+The above iformation is also covered in the [usage README](usage.md#--reads).
+
+
+
+### Some of my samples are missing!
+If the pipeline ran witout crashing and some samples are missing then there are two possible expalations:
```
Comment (ewels) [inline docs/Troubleshooting.md]: expalations -> explanations

Comment (ewels) [reply docs/Troubleshooting.md]: witout -> without

File: docs/Troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+1. You made a misstake when declaring the input files - in that case see above.
```
Comment (ewels) [inline docs/Troubleshooting.md]: misstake -> mistake

File: docs/Troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+
+
+### Some of my samples are missing!
+If the pipeline ran witout crashing and some samples are missing then there are two possible expalations:
+
+1. You made a misstake when declaring the input files - in that case see above. 
+2. Your sample didn't reach 5 % read alignment.
+
+Samples that recive less than 5% alignment are skipped for further analysis. You will thus see STAR/Hisat2 output for such samples but nothing more. This limit was set for two reasons, one it doesn't really make any sense to waste CPU hours runnnig on data that bad. Secondly, and most importantly we found that some of the downstream processes would occationaly crash if we allowed it.
```
Comment (ewels) [inline docs/Troubleshooting.md]: runnnig -> running
occationaly -> occasionally
